readconfig_test.go

```diff
@@ -76,12 +77,18 @@ func Test_ReadConfig(t *testing.T) {
 		t.Fail()
 	}
 
-	expected = "test_gatewayaddr"
+	expected = "test_gatewayaddr:8080"
```

Comment:
Can you use the word `want` instead of `expected` and got` instead of `actual`? Thank you, see: https://blog.alexellis.io/golang-writing-unit-tests/

Reply:
I think we should then refactor the whole test file, since all tests are using `expected` and `actual` and we should do this in another PR. Do you agree?

Reply:
Yes, but this is not refactoring, it's naming. Please go ahead.